style.scss

```diff
@@ -1,2 +1,119 @@
-// @import "@fortawesome/fontawesome-free/css/all.css";
-// @import "node_modules/bootstrap/scss/bootstrap";
+@import "@fortawesome/fontawesome-free/css/all.css";
+@import "node_modules/bootstrap/scss/bootstrap";
+
+
+#primary_nav ul {
+  list-style: none;
+  background: -webkit-linear-gradient(rgb(63, 65, 65), rgb(90, 92, 92));
+  opacity: 80%;
+  padding: 5px 0;
+}
+
+#primary_nav li a {
+  display: block;
+  color: #fff;
+  text-decoration: none;
+  margin-left: 60%;
+  width: 100%;
+  display: block;
+}
+
+#primary_nav li a:hover {
+  background: #ccc;
+  width: 100%;
+}
+.menu {
+  display: block;
+  float: right;
+  width: 40px;
+  height: 40px;
+  margin-top: 10%;
+}
+@media only screen and (min-width: 768px) {
+  .menu {
+    display: none;
+  }
+
+  .wrapper {
+    position: relative;
+    width: 768px;
+   
+  }
+
+  #primary_nav {
+    width: 100%;
+    color: white;
+  }
+  #primary_nav ul {
+    color: white;
+
+  }
+
+  #primary_nav ul li {
+    text-align: center;
+    position: relative;
+    margin: auto;
+  }
+
+  #primary_nav ul::after {
+    content: "";
+    display: table;
+    clear: both;
+  }
+
+  #primary_nav li a {
+    display: block;
+    color: #fff;
+    text-decoration: none;
+    margin-left: 60%;
+    width: 100%;
+    display: block;
+  }
+  #primary_nav a {
+    text-decoration: none;
+    color: white;
+    text-align: center;
+  }
+  .sous {
+    display: none;
+    background-color: white;
+    position: absolute;
+    width: 100%;
+
+  }
+  #primary_nav > ul li:hover .sous {
+    display: block;
+  }
+  .sous li {
+    width: 100%;
+    text-align: left;
+  }
+  .sous a:hover {
+    background-color: RGBa(200, 200, 200, 0.1);
+  }
+  .deroulant > a::after {
+    content: " ▼";
+    font-size: 10px;
+  }
```

Comment:
Sorry again, I don't understand that part (line 94 to 97)... 
Needs comments :)